synapse/config/tls.py

```diff
@@ -70,6 +72,50 @@ def read_config(self, config):
 
         self.tls_fingerprints = list(self._original_tls_fingerprints)
 
+        # Whether to verify certificates on outbound federation traffic
+        self.federation_verify_certificates = config.get(
+            "federation_verify_certificates", False,
+        )
+
+        # Whitelist of domains to not verify certificates for
+        fed_whitelist_entries = config.get(
+            "federation_certificate_verification_whitelist", [],
+        )
+
+        # Support globs (*) in whitelist values
+        self.federation_certificate_verification_whitelist = []
```

Comment:
for future reference, you could write this:

```
self.federation_certificate_verification_whitelist = [glob_to_regex(entry) for entry in fed_whitelist_entries]
```

what you have is fine though.